app/pulp/app/settings.py

```diff
@@ -113,9 +112,11 @@
     'DEFAULT_FILTER_BACKENDS': ('rest_framework.filters.DjangoFilterBackend',),
     'DEFAULT_PAGINATION_CLASS': 'pulp.app.pagination.UUIDPagination',
     'PAGE_SIZE': 100,
-    'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAuthenticated',),
 }
 
+if not DEBUG:
+    REST_FRAMEWORK['DEFAULT_PERMISSIONS_CLASSES'] = ('rest_framework.permissions.IsAuthenticated',)
```

Comment:
On second though, this makes me a little uncomfortable, so I think I'm going to just pull it out of this PR and also capture this in its own story/task, and also add a note about how to easily disable auth on the api for lazy developing.